### PR TITLE
Fix panic in generating `emit` for incomplete signal

### PR DESCRIPTION
**Pull request by EPashkin**
Fix panic on GObject generation (see https://github.com/gtk-rs/gir/pull/620#issue-199102491).

cc @sdroege

### CODE REVIEW

**Comment by EPashkin** [comment]
Also can be fixed by just adding "GObject.ParamSpec" to manual in Gir_GObject.toml,
but with this we can find what type need be deignored

File: src/codegen/signal.rs [hunk 1]
```diff
@@ -75,6 +75,11 @@ pub fn generate(
         }
     }
 
+    if function_type.is_none() {
+        // Signal incomplete, can't generate emit
+        return Ok(());
```
Comment (sdroege) [inline src/codegen/signal.rs]: Makes sense, but which signal is that exactly?

Comment (sdroege) [reply src/codegen/signal.rs]: I see, makes sense

**Comment by EPashkin** [comment]
Ok, let merge this then

**Pull request merged**